packages/contract/script/Constants.s.sol

```diff
@@ -352,34 +389,43 @@ function _getTokenMinterAddress(uint256 chainId) pure returns (address) {
     revert("Unsupported chainID");
 }
 
+// ----------------- Across ----------------- //
+
+address constant ETH_MAINNET_SPOKE_POOL = 0x5c7BCd6E7De5423a257D81B442095A1a6ced35C5;
+address constant OP_MAINNET_SPOKE_POOL = 0x6f26Bf09B1C792e3228e5467807a900A503c0281;
+address constant ARBITRUM_MAINNET_SPOKE_POOL = 0xe35e9842fceaCA96570B734083f4a58e8F7C5f2A;
+address constant BASE_MAINNET_SPOKE_POOL = 0x09aea4b2242abC8bb4BB78D537A67a245A7bEC64;
+address constant POLYGON_MAINNET_SPOKE_POOL = 0x9295ee1d8C5b022Be115A2AD3c30C72E34e7F096;
+address constant LINEA_MAINNET_SPOKE_POOL = 0x7E63A5f1a8F0B4d0934B2f2327DAED3F6bb2ee75;
+address constant ZKSYNC_MAINNET_SPOKE_POOL = 0xE0B015E54d54fc84a6cB9B666099c46adE9335FF;
+address constant BLAST_MAINNET_SPOKE_POOL = 0x2D509190Ed0172ba588407D4c2df918F955Cc6E1;
+address constant SCROLL_MAINNET_SPOKE_POOL = 0x3baD7AD0728f9917d1Bf08af5782dCbD516cDd96;
+address constant ZORA_MAINNET_SPOKE_POOL = 0x13fDac9F9b4777705db45291bbFF3c972c6d1d97;
+address constant MODE_MAINNET_SPOKE_POOL = 0x3baD7AD0728f9917d1Bf08af5782dCbD516cDd96;
+address constant LISK_MAINNET_SPOKE_POOL = 0x9552a0a6624A23B848060AE5901659CDDa1f83f8;
+address constant REDSTONE_MAINNET_SPOKE_POOL = 0x13fDac9F9b4777705db45291bbFF3c972c6d1d97;
```

Comment:
spoke pool address same on Redstone and Zora?

Reply:
yeah, they're the same